deploy/roles/gocd/meta/main.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 dependencies:
   - {role: python, when: "install_python and custom_repo"}
-  - {role: python-build, when: "install_python and not custom_repo"}
+  - {role: python-build, when: "not custom_repo"}
```

Comment:
@tanderegg We only need to skip a few python tasks to avoid the 10 minute limitation in travis. So rather than skipping the whole role, I thought it made sense to only skip the tasks that take a bit to install.